### PR TITLE
[Meghana-Step 8] Add system spec

### PR DESCRIPTION
**Pull request by Meghana-Reddy2**
Description 

Add rspec & system spec for tasks 

Screenshot 2024-02-27 at 13 29 55
Screenshot 2024-02-27 at 13 30 30
Screenshot 2024-02-27 at 13 30 41
Screenshot 2024-02-27 at 13 30 55

### CODE REVIEW

File: myapp/spec/system/tasks_spec.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      end
+  
+      it 'task deleted successfully' do
+        visit '/'
+        expect(Task.all.length).to eq 1
+        click_on "Delete"
+        expect(Task.all.length).to eq 0
+      end
+    end
+  end
```
Comment (yo-it-engineer) [inline myapp/spec/system/tasks_spec.rb]: image

not a big deal but it's a good practice to add a new line after the final line of the file.

**Review by yo-it-engineer — CHANGES_REQUESTED** [review]
I was surprised to see so many lines changed!
`myapp/log/development.log` is not needed. 😅
I think you can remove it from Git.

File: myapp/spec/system/tasks_spec.rb (hunk 1 above)
Comment (Meghana-Reddy2) [reply myapp/spec/system/tasks_spec.rb]: I added "log/*.log" line to gitignore file. But I think its still not getting ignored.

Comment (yo-it-engineer) [reply myapp/spec/system/tasks_spec.rb]: after adding the line into gitignore file, you still need to manually remove the tracking of the file that you have checked in with git.

image

I have run the process and committed it. can you pull it and check that if git still show you difference for `log/development.log` file?

Comment (Meghana-Reddy2) [reply myapp/spec/system/tasks_spec.rb]: > I added "log/*.log" line to gitignore file. But I think its still not getting ignored.

Its removed now. and I fixed the typos also

**Comment by Meghana-Reddy2** [comment]
Its removed now. and I fixed the typos also

**Review by yo-it-engineer — APPROVED** [review]
The rest LGTM!

**Pull request merged**